spec/system/jobseekers/school_search_mobile_display_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+RSpec.describe "Jobseeker job search filter mobile display" do
+  let(:request) { visit organisations_path(params) }
```

Comment:
Could this not just be in the 'before' block? Does the 'visit' method actually return anything meaningful?